openspoor/visualisations/trackmap.py

```diff
@@ -93,35 +109,44 @@ def _fix_zoom(self) -> None:
             bounds = [min(map(lambda x: x[i], bboxes)) if i < 2 else max(map(lambda x: x[i], bboxes)) for i in range(4)]
             self.fit_bounds([bounds[:2], bounds[2:]])
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
+    def add(self, plotobject):
+        plotobject.add_to(self)
+
+    def show(self, makesmaller: bool = False):
         """
-        End the context manager, keeping track of any exceptions that may have occurred.
-        :return: A SpoorKaart object
+        Fix the map so it is zoomed to a nice level and is displayed at a suitable size.
+
+        :param makesmaller: A boolean for decreasing the size of the final plot. Designed for Databricks Notebooks
+        :return:
         """
-        if exc_type is not None:
-            raise exc_type(exc_val)
         self._fix_zoom()
-        if self.save_name is not None:
-            print(f'Saving as {self.save_name}')
-            self.save(self.save_name)
-        if self.makesmaller:
+        if makesmaller:
             self.superfigure = folium.Figure(1000, 400)
             self.add_to(self.superfigure)
             return self.superfigure
         else:
             return self
 
+    def save(self, save_name: Path):
+        """
```

Comment:
4 spaces tabs.. thats not PEP8 :-)

Reply:
This happens throughout the whole repo, left as is for now. Will note this as an issue.